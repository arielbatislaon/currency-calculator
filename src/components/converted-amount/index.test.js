import React from 'react';
import renderer from 'react-test-renderer';

import ConvertedAmount from './index';

it('renders ConvertedAmount correctly ', () => {
  const tree = renderer.create(<ConvertedAmount />).toJSON();
  expect(tree).toMatchSnapshot();
});