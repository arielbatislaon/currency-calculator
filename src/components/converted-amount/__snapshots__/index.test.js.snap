// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ConvertedAmount correctly  1`] = `
<label
  className="result"
  id="result"
>
  <div
    aria-labelledby="result"
    className="label"
  >
    Converted Amount:
  </div>
  <input
    aria-labelledby="result"
    disabled={true}
    type="text"
  />
</label>
`;
